packages/terra-application-links/src/tabs/ApplicationTabs.module.scss

```diff
@@ -172,11 +172,13 @@
       word-break: normal;
       word-wrap: normal;
     }
-
-    &:hover,
-    &.is-hovered {
-      .tab-inner {
-        background-color: var(--terra-application-links-tabs-tab-hover-background-color, rgba(0, 0, 0, 0.15));
+    /* stylelint-disable max-nesting-depth */
```

Comment:
does this need to be re-enabled or can we only disable the next line?

Reply:
Disabled for next line only 0c9bba3